### ISSUE TITLE
Migration to web‐api v7 - Unexpected `token` argument in `apps.uninstall` method

### ISSUE THREAD

**Issue by tristan-s** [slackapi/node-slack-sdk#1871]
We are transitioning our application using the v6 of the `@slack/web-api` package to the v7. One breaking change is detected by the compiler, related to the `apps.uninstall` method.

Previously, our wrapper defined the uninstall methods like this:
```ts
async uninstallApplication() {
    const accessToken = await this.accessTokenFetcher.getAccessToken();

    const { error } = await this.slackClient.apps.uninstall({
        token: accessToken,
        client_id: _credentials.slack.clientId,
        client_secret: _credentials.slack.clientSecret
    });
    if (error) { throw new SlackResponseError(error); }
}
``` 

Now, we get an error on the `token` parameter of that method, which suggests that the `token` field is not expected by the `apps.uninstall` method:

```
Vue: Object literal may only specify known properties, and token does not exist in type
Pick<OAuthCredentials, "client_id" | "client_secret">
```

We've checked the [doc for apps.uninstall](https://api.slack.com/methods/apps.uninstall), and `token` is still listed as a required argument.

Is the documentation outdated? Or are we misusing the `apps.uninstall` method?

#### Packages:

Select all that apply:

- [x] `@slack/web-api`
- [ ] `@slack/rtm-api`
- [ ] `@slack/webhooks`
- [ ] `@slack/oauth`
- [ ] `@slack/socket-mode`
- [ ] `@slack/types`
- [ ] I don't know

#### The Slack SDK version

7.3.2

#### Node.js runtime version

v20.15.1

#### OS info

ProductName:		macOS
ProductVersion:		14.5
BuildVersion:		23F79
Darwin Kernel Version 23.5.0: Wed May  1 20:13:18 PDT 2024; root:xnu-10063.121.3~5/RELEASE_ARM64_T6030

#### Steps to reproduce:

1. Update to the version 7.3.2 of the `web-api` package
2. Try to call `slackClient.apps.uninstall` with a `token` argument

### Expected result:

- Either the documentation and the code both say we don't need a `token` argument;
- Or we don't get an error

### Actual result:

<img width="554" alt="image" src="https://github.com/user-attachments/assets/26caef7e-47c0-4051-904f-64897843c6ba">

### Requirements

For general questions/issues about Slack API platform or its server-side, could you submit questions at https://my.slack.com/help/requests/new instead. :bow:

Please read the [Contributing guidelines](https://github.com/slackapi/node-slack-sdk/blob/main/.github/contributing.md) and [Code of Conduct](https://slackhq.github.io/code-of-conduct) before creating this issue or pull request. By submitting, you are agreeing to those rules.

**Comment by zimeg** [comment]
Hey @tristan-s 👋  Thanks for raising this - I can confirm this causes error! It does seem unexpected and I'm noticing both the `apps.uninstall` and `admin.apps.uninstall` methods are missing this token override too...

I think both of these methods should offer the `token` attribute for the exact case you're outlining, so I'll get started on a fix 👀

As a quick workaround I'd suggest using `.apiCall('apps.uninstall', { ... })` from the [using a dynamic method name](https://slack.dev/node-slack-sdk/web-api/#call-a-method) section to avoid type-checking if this is blocking you right now!

**Comment by zimeg** [comment]
@tristan-s thanks again for sharing this! A fix should now be available in [`@slack/web-api@7.3.3`](https://github.com/slackapi/node-slack-sdk/releases/tag/%40slack%2Fweb-api%407.3.3)! 🚀
